minvws/nl-kat-coordination
# Implement boefje details modal in report config flow

### Changes

Add modal component to plugin tile, to show boefje details in a pop-up window

### Issue link

Closes #2969 

### Demo
<img width="1014" alt="Screenshot 2024-08-15 at 16 55 31" src="https://github.com/user-attachments/assets/02705754-4961-4c37-96a6-5a3076dabfae">
 
### QA notes

Make a report and check in step "Configuration" that "see details" on a plugin tile now opens a modal dialog with detailed info, instead of it being a link to the boefje detail page.

---

### Code Checklist






- [x] Tickets have been created for newly discovered issues: #3375

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
